applications/sasquatch/values-idfdev.yaml

```diff
@@ -60,6 +64,8 @@ chronograf:
   ingress:
     enabled: true
     hostname: data-dev.lsst.cloud
+  image:
+    tag: "1.10.1"
```

Comment:
We should be using 1.9.4 everywhere, not critical for dev, but we don't need this change.

Reply:
OK.  We had had 1.10.1 running in dev, and I didn't want to change it.